src/components/WhoWeAreSection.jsx

```diff
@@ -1,7 +1,48 @@
 import React from "react";
-
+import { FaGear } from "react-icons/fa6";
 const WhoWeAreSection = () => {
-  return <div>WhoWeAreSection</div>;
+  return (
+    <>
+      <div className="flex  relative">
+        <div className="flex flex-col">
+          <div className="w-40 h-32">
+            <div className="bg-[#6BABCF] w-32 h-3"></div>
+            <div className="bg-white absolute mt-2 mx-28 px-4 text-[#38769F] font-poppins font-bold p-1 text-2xl inline-flex items-center">
```

Comment:
```suggestion
            <div className="bg-white absolute mt-2 mx-28 px-4 text-[#38769F] font-poppins font-bold p-1 text-2xl inline-flex items-center">
```

Use the Tailwind Colors:

ex. `bg-asme-blue-100`